app/src/standard/java/com/x8bit/bitwarden/data/platform/manager/CrashLogsManagerImpl.kt

```diff
@@ -20,6 +21,12 @@ class CrashLogsManagerImpl(
         set(value) {
             settingsRepository.isCrashLoggingEnabled = value
             Firebase.crashlytics.isCrashlyticsCollectionEnabled = value
+            if (value) {
+                Timber.plant(Timber.DebugTree())
```

Comment:
I'm a little hesitant to plant DebugTree in non-debug builds. My concern is that we're going to end up logging more than crashes to logcat when `isCrashLoggingEnabled` in production without the users explicit knowledge. How do you feel about only using the `NonfatalErrorTree` when `isCrashLoggingEnabled`, and plant the `DebugTree` only in debug builds, regardless of the crash logging setting state? We would make sure `DebugTree` is not uprooted when toggling `isCrashLoggingEnabled` off on debug builds.